src/20250527_Multi_AssetParametersOptimization/AssetParametersOptimization.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Asset Parameters Optimization"
+author: "Aave-Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-asset-parameters-optimization"
```

Comment:
shouldn't the link be https://governance.aave.com/t/direct-to-aip-asset-parameters-optimization/22178 ?